Gifski/ConversionCompletedView.swift

```diff
@@ -0,0 +1,118 @@
+import Cocoa
+
+class ConversionCompletedView: SSView {
+	var fileUrl: URL? {
+		didSet {
+			draggableFile.fileUrl = fileUrl
+			if let url = fileUrl {
+				draggableFile.image = NSWorkspace.shared.icon(forFile: url.path)
+				fileName = url.lastPathComponent
+
+				showInFinderButton.onAction = { _ in
+					NSWorkspace.shared.activateFileViewerSelecting([url])
+				}
+			}
+		}
+	}
+
+	var fileName: String = "" {
+		didSet {
+			fileNameLabel.text = fileName
+		}
+	}
+
+	var fileSize: String = "" {
+		didSet {
+			fileSizeLabel.text = fileSize
+		}
+	}
+
+	private let fileNameLabel = with(Label()) {
+		$0.textColor = NSColor.secondaryLabelColor
+	}
+
+	private let fileSizeLabel = with(Label()) {
+		$0.textColor = NSColor.secondaryLabelColor
+	}
+
+	public let draggableFile = DraggableFile()
+
+	lazy var showInFinderButton = with(CustomButton()) {
+		$0.title = "Show in Finder"
+		$0.textColor = .appTheme
+		$0.backgroundColor = .clear
+		$0.borderWidth = 1
+	}
+
+	override init(frame: NSRect) {
+		super.init(frame: frame)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	public func updateFileSize() {
+		let formatter = ByteCountFormatter()
+		formatter.zeroPadsFractionDigits = true
+
+		var size: UInt64 = 0
+
+		do {
+			let attr = try FileManager.default.attributesOfItem(atPath: fileUrl!.path)
+			let dict = attr as NSDictionary
+			size = dict.fileSize()
+		} catch {
+			print("Error: \(error)")
+		}
+
+		fileSize = formatter.string(fromByteCount: Int64(size))
+	}
```

Comment:
The logic for getting the pretty file size should be moved into a `URL` extension.